src/net/sourceforge/kolmafia/swingui/FamiliarTrainingFrame.java

```diff
@@ -1576,9 +1588,11 @@ private void checkAvailableEquipment(final LockableListModel<AdventureResult> in
       if (this.familiar.getId() == FamiliarPool.CHAMELEON
           || this.leadNecklace
               && this.ratHeadBalloon
+              && this.petSweater
```

Comment:
placed twice

Reply:
Well, poo. I was originally putting it in front of pumpkin bucket but moved it to in front of sugar shield.

Reply:
Fixed.